src/routes/auth.routes.js

```diff
@@ -0,0 +1,19 @@
+import { signupValidation, loginValidation } from '../utils/authentication';
+
+import express from 'express';
+import {
+  signup,
+  login,
+  changePassword,
+  deleteUser,
+} from '../controllers';
+
+
+const router = express.Router();
+
+router.route('/signup').post(signupValidation, signup);
+router.route('/login').post(loginValidation, login);
+router.route('/changePassword').post(changePassword);
+router.delete('/deleteUser').delete(deleteUser);
```

Comment:
this should be router.route

Reply:
actually im not so sure about the naming of this endpoint '/deleteUser'. is this a common practice?

Reply:
That was not the problem

Reply:
yeah i know but I think it should be something like users/delete or some other name. we can think about it later

Reply:
oh the router.delete should still be router.route. you are applying delete methods twice. did u check if deleting user is working?